File: 3346.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
const pegarValores = (line) => line.split(" ").map(a => Number(a));

let [a, b] = pegarValores(lines.shift())

let flutuacao = ((((1.0 + a/100.00) * (1.0 + b/100.00)) - 1.0) * 100.0)

console.log(flutuacao.toFixed(6));

File: 3346.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String[] line = sc.nextLine().trim().split(" ");
        double a = Double.parseDouble(line[0]);
        double b = Double.parseDouble(line[1]);

        double flutuacao = (((1.0 + a / 100.00) * (1.0 + b / 100.00)) - 1.0) * 100.0;

        System.out.println(String.format("%.6f", flutuacao));
    }
}
